aptos-move/move-examples/token_objects/ambassador/move/Move.toml

```diff
@@ -3,7 +3,7 @@ name = 'ambassador_token'
 version = '1.0.0'
 
 [addresses]
-token_objects = "_"
+token_objects = "0xABCD"
```

Comment:
in order for me to compile this, I cannot have placeholder here, but an actual address (that gets replaced in the compiled objects later with correct address)

let me know if you have any concerns here

Reply:
I think this should be fine.